geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/RegionAdvisor.java

```diff
@@ -834,7 +834,7 @@ public InternalDistributedMember adviseFixedPrimaryPartitionDataStore(final int
     });
   }
 
-  Set adviseAllServersWithInterest() {
+  public Set adviseAllServersWithInterest() {
```

Comment:
No need to make it `public`, it's only invoked from `PRTombstoneMessage`, which belongs to the same package.